dist/rpm/openQA.spec

```diff
@@ -576,6 +577,7 @@ fi
 
 %files common
 %dir %{_datadir}/openqa
+%{_datadir}/openqa/script/openqa-check-devel-repo
```

Comment:
Hm, thinking: Putting this into "common" means that this will be pulled in with the "scripts" package, the webUI package as well as the worker package. So far the "common" package contains what I consider really common for all, but this script has a rather specific purpose. How about just keeping this in auto-update and you would not need the auto-update package to pull in common and then within *our* workers where we want to use this script we also install openQA-auto-update explicitly even though we don't use the systemd file there but only the script explicitly. WDYT?

Reply:
By the way. This change is included in the git commit "Improve health-check for devel:openQA repository" which does not sound like a good title for that. "Extract health check …" would be better in that case